provider/google_test.go

```diff
@@ -523,18 +581,9 @@ func createZone(t *testing.T, provider *GoogleProvider, zone *dns.ManagedZone) {
 func setupGoogleRecords(t *testing.T, provider *GoogleProvider, endpoints []*endpoint.Endpoint) {
 	clearGoogleRecords(t, provider, "zone-1-ext-dns-test-2-gcp-zalan-do")
 	clearGoogleRecords(t, provider, "zone-2-ext-dns-test-2-gcp-zalan-do")
-
-	records, err := provider.Records()
-	require.NoError(t, err)
-
-	validateEndpoints(t, records, []*endpoint.Endpoint{})
```

Comment:
why was this check removed ?

Reply:
This is a mistake.
Thanks so much for catching it! 👍

Reply:
Removed.